### PR TITLE
Setter ignore for LCOM decomposition

### PR DESCRIPTION
**Pull request by lyriccoder**
Please specify the following in the description:

1. https://github.com/cqfn/aibolit/issues/617
2. Now we can ignore setter in AST decomposition, if we want.

a) Setter pattern was written in a wrong way: it checked whether function name equals the filed name the following way:
field `bitMap`, function `setBitMap` ->setter
field `bitMap`, function `setBlabla { this bitmap=0; }` ->not a setter.
It was fixed and new cases were found, like the following one:
```
  public void setRestrictSystemProperties(boolean val) {
    this.restrictSystemProps = val;
  }
```

It was found in a real example.

b) HybridConstructor module name was renamed. There was just a wrong name
c) Some tests were added for AST decompostion with setters

### CODE REVIEW

File: test/patterns/classic_setter/test_classic_setter.py [hunk 1]
```diff
@@ -43,11 +43,14 @@ def test_four_setter_patterns(self):
         ast = AST.build_from_javalang(build_ast(filepath))
         pattern = ClassicSetter()
         lines = pattern.value(ast)
-        self.assertEqual(lines, [1236, 1240, 3783, 3819])
+        self.assertEqual(lines,
```
Comment (aravij) [inline test/patterns/classic_setter/test_classic_setter.py:46]: Why here and in next test values changed?

**Review by aravij — APPROVED** [review]

File: test/patterns/classic_setter/test_classic_setter.py (hunk 1 above)
Comment (lyriccoder) [reply test/patterns/classic_setter/test_classic_setter.py:46]: I have explained it in the description:
 Pattern checked whether function name equals the filed name the following way:
field bitMap, function setBitMap ->setter

But now i just check if only the function begins with `set`

Is it clearer now?

**Comment by lyriccoder** [comment]
@acheshkov  merge it

**Review by acheshkov — APPROVED** [review]

**Comment by acheshkov** [comment]
@rultor merge

**Comment by acheshkov** [comment]
@rultor hello

**Comment by acheshkov** [comment]
Test

**Comment by lyriccoder** [comment]
@acheshkov do it again pls. @rultor ignores you

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**